src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -2686,6 +2686,11 @@ protected AssemblySymbol GetForwardedToAssembly(string name, int arity, ref Name
         }
 
 #nullable enable
+        internal static bool IsFeatureAvailable(SyntaxNode syntax, MessageID feature)
```

Comment:
Could we use the existing `CSharpCompilationExtensions.IsFeatureEnabled(this CSharpCompilation, MessageID)` instead of introducing a new helper?

Reply:
Yes, thanks.